content/en/docs/reference/glossary/manifest.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Manifest
+id: manifest
+date: 2019-06-28
+short_description: >
+  Definition of a Kubernetes API object, serialized into a configuration file (eg JSON or YAML)
```

Comment:
I'd almost simplify this further to just

"A file containing the serialized definition of one or more Kubernetes API objects"

I don't think it's particularly useful to say "configuration file" versus just "file".